test/screenplay/suppressed_predictions_test.exs

```diff
@@ -0,0 +1,324 @@
+defmodule Screenplay.SuppressedPredictionsTest do
+  alias Screenplay.PlaceCacheHelpers
+  alias Screenplay.SuppressedPredictions
+  alias Screenplay.SuppressedPredictions.SuppressedPrediction
+  use ScreenplayWeb.DataCase
+
+  import Screenplay.Factory
+
+  setup_all do
+    start_supervised!(Screenplay.Places.Cache)
+    :ok
+  end
+
+  describe "suppressed_predictions" do
+    setup do
+      PlaceCacheHelpers.seed_place_cache()
+      :ok
+    end
+
+    test "get_suppressed_predictions/1" do
+      insert(:suppressed_predictions, %{
+        id: 1,
+        location_id: "place-zero",
+        direction_id: 1,
+        clear_at_end_of_day: true,
+        inserted_at: ~U[2024-05-01T01:00:00Z],
+        updated_at: ~U[2024-05-01T01:00:00Z]
+      })
+
+      insert(:suppressed_predictions, %{
+        id: 2,
+        location_id: "place-one",
+        direction_id: 2,
+        clear_at_end_of_day: true,
+        inserted_at: ~U[2024-05-01T01:00:00Z],
+        updated_at: ~U[2024-05-01T01:00:00Z]
+      })
+
+      insert(:suppressed_predictions, %{
+        id: 3,
+        location_id: "place-two",
+        direction_id: 0,
+        clear_at_end_of_day: true,
+        inserted_at: ~U[2024-05-01T01:00:00Z],
+        updated_at: ~U[2024-05-01T01:00:00Z]
+      })
+
+      assert %SuppressedPrediction{
+               id: 3,
+               location_id: "place-two",
+               direction_id: 0,
+               clear_at_end_of_day: true
+             } =
+               SuppressedPredictions.get_suppressed_prediction(3)
+    end
```

Comment:
There's a fair bit of test code here that seems like it's just testing basic Ecto operations, so there's not many ways for this to fail due to a bug in our app code. Given that we have controller tests that are exercising this same functionality but in a more integrated, "real" usage pattern, we could probably slim down these low-level tests without losing much confidence.